lib/tasks/salt.rake

```diff
@@ -5,6 +5,19 @@ namespace :salt do
       puts "Please provide WORKER_ID environment variable that identifies this worker"
       Process.exit 1
     end
+    if ENV["SALT_MINION_RECONCILER"] == "true"
+      # rubocop:disable Lint/HandleExceptions
+      Thread.new do
+        loop do
```

Comment:
This rake task will be called only once, and the thread is kept running because https://github.com/kubic-project/velum/pull/482/files#diff-c63c488acc269ddcd79fdb857d5b87d6R21 will be running also in a loop. There's not even need to join the thread in this case, because both are infinite loops.